### ISSUE TITLE
Could not pass event AsyncPlayerChatEvent to EssentialsChat

### ISSUE THREAD

**Issue by mibby** [EssentialsX/Essentials#721]
Essentials dev b370
EssentialsChat dev b370
Paper dev 810

```
[16:46:45] [Async Chat Thread - #8/WARN]: [Essentials] Permissions lag notice with (PermissionsExHandler). Response took 177.656379ms. Summary: Getting group for <player>
[16:46:45] [Async Chat Thread - #8/ERROR]: Could not pass event AsyncPlayerChatEvent to EssentialsChat v2.0.1-b370
org.bukkit.event.EventException
    at org.bukkit.plugin.EventExecutor$1.execute(EventExecutor.java:46) ~[patched_1.10.2.jar:git-Paper-810]
    at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:74) ~[patched_1.10.2.jar:git-Paper-810]
    at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.10.2.jar:git-Paper-810]
    at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:517) [patched_1.10.2.jar:git-Paper-810]
    at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:499) [patched_1.10.2.jar:git-Paper-810]
    at net.minecraft.server.v1_10_R1.PlayerConnection.chat(PlayerConnection.java:1304) [patched_1.10.2.jar:git-Paper-810]
    at net.minecraft.server.v1_10_R1.PlayerConnection.a(PlayerConnection.java:1219) [patched_1.10.2.jar:git-Paper-810]
    at net.minecraft.server.v1_10_R1.PacketPlayInChat$1.run(PacketPlayInChat.java:39) [patched_1.10.2.jar:git-Paper-810]
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_92]
    at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_92]
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [?:1.8.0_92]
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [?:1.8.0_92]
    at java.lang.Thread.run(Thread.java:745) [?:1.8.0_92]
Caused by: java.lang.UnsupportedOperationException: Not supported yet.
    at com.earth2me.essentials.OfflinePlayer.getScoreboard(OfflinePlayer.java:1162) ~[?:?]
    at com.earth2me.essentials.chat.EssentialsChatPlayerListenerLowest.onPlayerChat(EssentialsChatPlayerListenerLowest.java:44) ~[?:?]
    at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor496.execute(Unknown Source) ~[?:?]
    at org.bukkit.plugin.EventExecutor$1.execute(EventExecutor.java:44) ~[patched_1.10.2.jar:git-Paper-810]
    ... 12 more
```

**Comment by SupaHam** [comment]
@vemacs any idea why all of these methods aren't noop? I've had to change some one by one and I'm wondering why not make all unsupported methods just noop?

**Comment by vemacs** [comment]
I really have no idea, I think we just rolled with the original Ess style of throwing UnsupportedOperationException without thinking it through too much

**Comment by SupaHam** [comment]
Should I NOOP them?

**Comment by vemacs** [comment]
Yeah

**Comment by SupaHam** [comment]
This has been fixed in 1423151. You can grab the fix from [build 376](https://ci.drtshock.net/job/EssentialsX/) on the CI server.
